app/src/main/java/ua/kozlov/madfood/Foods.java

```diff
@@ -0,0 +1,261 @@
+package ua.kozlov.madfood;
+
+import android.content.Context;
+import org.xmlpull.v1.XmlPullParser;
+import org.xmlpull.v1.XmlPullParserException;
+import java.io.IOException;
+import java.util.ArrayList;
+
+public class Foods implements FoodsInterface {
+
+    @Override
+    public ArrayList<String> getFoodsList(final String group_name, final Context context) {
+        XmlPullParser parser = getParser(group_name, context);
+        ArrayList<String> foodsList = setFoodNamesToList(parser);
+        return foodsList;
+    }
+
+    @Override
+    public String getFoodName(final String groupId, final String id, final Context context) {
```

Comment:
@OlehKozlov why do you have args in this method but you don't use them in this method?